pkg/asset/machines/master.go

```diff
@@ -158,7 +158,6 @@ func (m *Master) Generate(dependencies asset.Parents) error {
 		}
 		azure.ConfigMasters(machines, clusterID.InfraID)
 	case baremetaltypes.Name:
-		// FIXME: baremetal
```

Comment:
This was introduced in a rebase,  before we created machines for baremetal.  We create them now, so this FIXME (and the one below,I think) aren't needed.